analysis/src/analysis.js

```diff
@@ -84,11 +88,20 @@ class Analysis {
 
       this.npm.prune().then(() => {
         return this.npm.install(attributes.owner, attributes.repo, attributes.version);
-      }).then(root => {
+      }).then(async(root) => {
```

Comment:
I'm a little confused about what `root` is here. What does `npm.install` return?

Reply:
It returns the location of the installed package.

Reply:
Renamed variables and added a comment here.